modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PageMemoryLazyAllocationTest.java

```diff
@@ -42,18 +42,13 @@
     /** */
     public static final String EAGER_REGION = "eagerRegion";
 
-    /** */
-    protected boolean client = false;
```

Comment:
Compilation failed - not all `client` usages were removed.

Reply:
Fixed